Fails to build fontconfig-git

OS: Windows 10 (1903) (x64)

Script fails on fontconfig-git

```
make[3]: Entering directory '/build/fontconfig-git/build-64bit/src'
lib -name:libfontconfig-1.dll -def:fontconfig.def -out:fontconfig.lib
make[3]: lib: Command not found
make[3]: *** [Makefile:986: fontconfig.lib] Error 127
```

The Msys environment doesn't appear to include a lib.
I tried using Microsoft's lib but that fails due to an invalid option.

[logs.zip](https://0x0.st/zg_U.zip)

Tried using Microsoft's (visual studio 2017) lib.exe and it gets slightly farther, but this time there's an error with MSys64's linker.

```
lib -name:libfontconfig-1.dll -def:fontconfig.def -out:fontconfig.lib
LINK: unknown option -- n
Try 'LINK --help' for more information.
make[3]: *** [Makefile:986: fontconfig.lib] Error 1
```
`lib.exe: 14.15.26732.1 (x64)`
`link.exe' April 25 2019 3:36:02am (no version information) in msys64/usr/bin.`

[logs.zip](https://0x0.st/zgLz.zip)

Just re-ran this again on a clean install:

```
Likely error (tail of the failed operation logfile):
make[3]: Entering directory '/build/fontconfig-git/build-32bit/src'
lib -name:libfontconfig-1.dll -def:fontconfig.def -out:fontconfig.lib
make[3]: lib: Command not found
make[3]: *** [Makefile:986: fontconfig.lib] Error 127
make[3]: Leaving directory '/build/fontconfig-git/build-32bit/src'
make[2]: *** [Makefile:541: all] Error 2
make[2]: Leaving directory '/build/fontconfig-git/build-32bit/src'
make[1]: *** [Makefile:599: all-recursive] Error 1
make[1]: Leaving directory '/build/fontconfig-git/build-32bit'
make: *** [Makefile:483: all] Error 2
make failed. Check D:/Build/FFMPEG/build/fontconfig-git/build-32bit/ab-suite.make.log
This is required for other packages, so this script will exit.
14:54:46   Creating diagnostics file...
```

[logs.zip](https://0x0.st/zgL2.zip)

--- a/media-autobuild_suite.bat
+++ b/media-autobuild_suite.bat
@@ -68,6 +68,26 @@ do if %%f lss 4 (
     exit
 )
 
+(
+    where lib.exe || ^
+    where cl.exe || ^
+    if DEFINED VSINSTALLDIR cd .
+) >nul 2>&1 && goto MSVCINSTALLED
+goto MSVCNOTINSTALLED
+:MSVCINSTALLED
+echo ----------------------------------------------------------------------
+echo. You are running in a MSVC environment ^(cl.exe or lib.exe detected^)
+echo. This is not supported.
+echo. Please run the script through a normal cmd.exe some other way.
+echo.
+echo. Detected Paths:
+where lib.exe 2>nul
+where cl.exe 2>nul
+echo %VSINSTALLDIR%
+pause
+exit
+:MSVCNOTINSTALLED
+
 set build=%instdir%\build
 if not exist %build% mkdir %build%
 

